src/EnergyPlus/PlantPipingSystemsManager.cc

```diff
@@ -5656,7 +5656,7 @@ namespace PlantPipingSystemsManager {
         this->Cur.CurAirTemp = state.dataEnvrn->OutDryBulbTemp;
         this->Cur.CurWindSpeed = state.dataEnvrn->WindSpeed;
         this->Cur.CurRelativeHumidity = state.dataEnvrn->OutRelHum;
-        this->Cur.CurIncidentSolar = state.dataEnvrn->BeamSolarRad;
+        this->Cur.CurIncidentSolar = state.dataEnvrn->BeamSolarRad * max(state.dataEnvrn->SOLCOS(3), 0.0);
```

Comment:
:+1: At the time, I really did think that this BeamSolarRad variable took into account the solar angle.  :shrug:

Reply:
Well, when I first started looking at this, my reaction was "so what's the problem here, how is it not correct?" but then eventually verified that it was a problem.